src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -310,6 +313,16 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
           }
         }
 
+        this.logService.createFeedbackSessionLog({
+          courseId: this.courseId,
+          studentEmail: this.loggedInUser,
+          logType: LogTypes.FEEDBACK_SESSION_ACCESS,
+        }).subscribe(() => {
+
+        }, () => {
+          this.simpleModalService.openInformationModal('Log Error', SimpleModalType.WARNING, '');
```

Comment:
Maybe would be better to use a toast instead?

@damithc If the tracking of student access/submission fails, would a modal or toast be more appropriate?

Reply:
changed to toast. I feel it makes more sense too. thanks for the feedback